### ISSUE TITLE
Less verbose output when loading multifile TIFFs

### ISSUE THREAD

**Issue by anntzer** [ome/bioformats#3949]
When converting a multifile TIFF (e.g. from a Metamorph .nd MDA acquisition) with bfconvert (or when opening it with fiji), the output can get many, many lines of "Reading IFDs"/"Populating metadata"/"Populating OME metadata", which is in fact obscuring the (more) interesting output (e.g. the progress from bfconvert "Series S: converted T/U planes (V%)").  Naturally, one can grep these repeated lines away, but perhaps consider not emitting them at all, or making them togglable via a command-line switch or environment variable?

**Comment by dgault** [comment]
Hi @anntzer, thank you for the feedback. The `bfconvert` tool in particular currently only has an option (-debug) to toggle the debug logging output, so you can't currently disable the output you are seeing. The `showinf` tool has a greater number of options for different levels of debugging (debug, info, trace and off). We should be able to add similar options to the bfconvert tool to give a greater level of control to the output.

**Comment by anntzer** [comment]
From a quick look, both the "Reading IFDs"/... logs and the progress report logs are made at INFO level (I think), so I'm not sure a global switch would be enough (it's still better than nothing, though).

**Comment by dgault** [comment]
Yeah I could see the progress reports being moved to a different level in this case.

**Comment by melissalinkert** [comment]
Suppressing these messages in the command line tools can be done by modifying the `logback.xml` file that is packaged with `bftools`, i.e. https://github.com/ome/bioformats/blob/develop/tools/logback.xml.

If you add this between lines 10 and 11:

```<logger name="loci.formats.in" level="WARN"/>```

that will switch just the readers to logging at `WARN`, but everything else would stay at `INFO`. Setting the name to a specific reader (e.g. `loci.formats.in.MinimalTiffReader`) or adding multiple `<logger ...>` lines for different classes at different levels is also a possibility. See https://logback.qos.ch/manual/configuration.html#syntax for more information about how to configure logback.
